lib/ansible/modules/network/avi/avi_cloud.py

```diff
@@ -210,7 +253,7 @@
       enable_vip_static_routes: false
       license_type: LIC_CORES
       mtu: 1500
-      name: vCenter Cloud
+      name: VCenter Cloud
```

Comment:
Lower-case `v` is correct here. See https://www.vmware.com/products/vcenter-server.html.
```suggestion
      name: vCenter Cloud
```